DiscountDiscover/API/APIManager.m

```diff
@@ -65,4 +49,28 @@ - (void)fetchDeals:(void(^)(NSArray *deals, NSError *error))completion {
     [task resume];
 }
 
++ (NSString *)getAPIKey:(NSString *)key {
+    NSString *path = [[NSBundle mainBundle] pathForResource:
+                    @"APIKeys" ofType:@"plist"];
+    NSDictionary *plist = [[NSDictionary alloc] initWithContentsOfFile:path];
+    return [plist valueForKey:key];
+}
+
++ (CLLocationCoordinate2D)getLocationCoordinate {
+    CLLocationManager *locationManager = [[CLLocationManager alloc] init];
+    locationManager.delegate = self;
+    locationManager.distanceFilter = kCLDistanceFilterNone;
+    locationManager.desiredAccuracy = kCLLocationAccuracyBest;
+    
+    if ([CLLocationManager authorizationStatus] != kCLAuthorizationStatusAuthorizedWhenInUse) {
+        [locationManager requestWhenInUseAuthorization];
+    }
+    
+    [locationManager startUpdatingLocation];
+    CLLocationCoordinate2D locationCoordinate = locationManager.location.coordinate;
```

Comment:
Not sure if this is the best way to do it, accuracy might be pretty bad if you check the location right after `startUpdatingLocation`. Have you checked the accuracy here?